Teacher Workout/Features/Authentication/SignIn/View/SignInView.swift

```diff
@@ -0,0 +1,33 @@
+import SwiftUI
+
+struct SignInView: View {
+    @ObservedObject var viewModel = SignInViewModel()
+
+    var body: some View {
+        GeometryReader { _ in
+            VStack(alignment: .leading) {
+                TrailingCloseButton {
+                    print("Closing sign in")
```

Comment:
If we present the login, is there a way to close it after we merge this PR?

Reply:
We need to implement landing page first, and then add the logic in the coordinator when closing the sign in view to show the onboarding flow.